less/_defaults/_font-config.less

```diff
@@ -0,0 +1,186 @@
+// Font declarations
+// --------------------------------------------------
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,400i,700');
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Font family
+// --------------------------------------------------
+@title-family: 'Open Sans', sans-serif;
+@body-family: 'Open Sans', sans-serif;
+@btn-family: 'Open Sans', sans-serif;
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Font weights
+// --------------------------------------------------
+@font-weight-light: 300;
+@font-weight-regular: 400;
+@font-weight-bold: 700;
+
+// 100	Thin (Hairline)
+// 200	Extra Light (Ultra Light)
+// 300	Light
+// 400	Normal
+// 500	Medium
+// 600	Semi Bold (Demi Bold)
+// 700	Bold
+// 800	Extra Bold (Ultra Bold)
+// 900	Black (Heavy)
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Line heights
+// --------------------------------------------------
+@title-line-height: 1.2;
+@body-line-height: 1.5;
+@btn-line-height: 1.5;
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Globals
+// --------------------------------------------------
+@body-size: 1rem;
+@body-weight: @font-weight-regular;
+
+@instruction-size: 1rem;
+@instruction-family: @body-family;
+@instruction-weight: @font-weight-regular;
+@instruction-line-height: @body-line-height;
+
+@btn-size: 1rem;
+@btn-weight: @font-weight-regular;
+
+@icon-size: 1.5rem;
+@icon-weight: @font-weight-regular;
+@icon-line-height: 1;
+
+@link-size: 1rem;
+@link-family: @body-family;
+@link-weight: @font-weight-regular;
+@link-line-height: @body-line-height;
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Menu header
+// --------------------------------------------------
+@menu-title-size: 3rem;
+@menu-title-family: @title-family;
+@menu-title-weight: @font-weight-regular;
+@menu-title-line-height: @title-line-height;
+
+@menu-subtitle-size: 2rem;
+@menu-subtitle-family: @title-family;
+@menu-subtitle-weight: @font-weight-light;
+@menu-subtitle-line-height: @title-line-height;
+
+@menu-body-size: 1.25rem;
+@menu-body-family: @body-family;
+@menu-body-weight: @font-weight-regular;
+@menu-body-line-height: @body-line-height;
+
+@menu-instruction-size: 1.25rem;
+@menu-instruction-family: @body-family;
+@menu-instruction-weight: @font-weight-regular;
+@menu-instruction-line-height: @body-line-height;
+// --------------------------------------------------
+
+// --------------------------------------------------
+// Menu item
+// --------------------------------------------------
+@menu-item-title-size: 1.25rem;
+@menu-item-title-family: @title-family;
+@menu-item-title-weight: @font-weight-bold;
```

Comment:
It's very minor but I've noticed a few inconsistencies with the application of title font weights. 

`@font-weight-bold` is only used by Menu item for example. It looks like `@font-weight-regular` should be the general title font weight and `@font-weight-light` for subtitles to keep it consistent?